tests/e2e/redis/basic-ingest-redis-serving.py

```diff
@@ -567,64 +587,63 @@ def test_large_volume_retrieve_online_success(client, large_volume_dataframe):
                 GetOnlineFeaturesRequest.EntityRow(
                     fields={
                         "customer_id": Value(
-                            int64_val=large_volume_dataframe.iloc[0][
-                                "customer_id"]
+                            int64_val=large_volume_dataframe.iloc[0]["customer_id"]
                         )
                     }
                 )
             ],
             feature_refs=feature_refs,
         )  # type: GetOnlineFeaturesResponse
-        is_ok = all([check_online_response(ref, large_volume_dataframe, response) for ref in feature_refs])
+        is_ok = all(
+            [
+                check_online_response(ref, large_volume_dataframe, response)
+                for ref in feature_refs
+            ]
+        )
         return None, is_ok
 
-    wait_retry_backoff(retry_fn=try_get_features,
-                       timeout_secs=90,
-                       timeout_msg="Timed out trying to get online feature values")
```

Comment:
This `wait_retry_backoff` is unreachable @mrzzy. Not sure if you have context on it?

Reply:
I will remove it for the time being. Please submit a new PR if it's actually being used.